### PR TITLE
[FEATURE slider] adds slider component

### PR DESCRIPTION
**Pull request by l2succes**
screen shot 2018-09-11 at 11 46 02 am

### CODE REVIEW

File: src/elements/Slider.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+export interface SliderProps extends BorderProps, SpaceProps {
+  className?: string
+  allowCross: boolean
+  disabled?: boolean
+  min: number
+  max: number
+  step: number
+  defaultValue: number[]
+  onAfterChange?: (min_max: [number, number]) => void
```
Comment (orta) [inline src/elements/Slider.tsx]: `min_max` -> `minMax` ?

Comment (l2succes) [reply src/elements/Slider.tsx]: yeah, agreed. I was hoping I could do `([min, max]: [number, number]) => void` but that doesn't work so I named it that way

**Comment by damassi** [comment]
haha @l2succes - I added this last night and updated Reaction! https://github.com/artsy/palette/pull/80 -- Your examples are more fleshed out so would you maybe want to add them to Range? Or should we kill Range and go with Slider?

**Comment by l2succes** [comment]
Damn @damassi, can't keep up with you 😜 

I say let's kill `Range` and go with `Slider`. I was gonna ping you about renaming `Slider` to `Carousel` in reaction

**Comment by l2succes** [comment]
No worries! Amazing work as always, updating this PR now

**Comment by damassi** [comment]
Also, really enjoying `react-powerplug`; have used it in a few spots now. Thanks @ds300 for introducing that that lib, it's quite handy 🔧

**Comment by l2succes** [comment]
Yeah thanks @ds300! didn't know this existed, very useful lib

**Comment by l2succes** [comment]
@damassi So I went ahead and renamed the components after all 🙃
https://github.com/artsy/reaction/pull/1237

**Pull request merged**